src/api.js

```diff
@@ -33,6 +34,7 @@ api.declare({
   method:      'put',
   route:       '/secret/:name(*)',
   name:        'set',
+  context:     ['prop1', 'prop2'],
```

Comment:
What's this about?  API methods don't have a `context` property, and there's nothing named `prop1` or `prop2` in this service..